### DISCUSSION TITLE
Configurable character limit on regex triggers

### DISCUSSION THREAD

**Discussion by PyrokineticDarkElf** [espanso/espanso#1253] [Feature requests and Ideas]
I would like to use larger character counts in my arguments, I'm a web dev and would like to use it to create anchors. For example:
```<a href="{{arg1}}" target="_blank">{{arg2}}</a>```
links and anchors will pretty much always be over 30 characters :/

Could it be configurable in the default.yml file?

**Comment by federico-terzi** [comment]
Hey @Staples1010,

Thank you for the feedback! I'll add it to the roadmap :)

Cheers :)

**Reply by suttonbradley** [reply]
Much appreciated if this gets done! I may have some extra time in the coming weeks and could look to add it if not.

**Comment by qlassiqa** [comment]
Almost a year now and this hasn't been added as a setting? :( 

I'd really love for this to be added as a configurable setting, because I have to rely on forms & a textbox to input a list of codes (form that steals the active window focus) instead of typing them as part of the command.

**Reply by federico-terzi** [reply]
@AucaCoyan I feel this could be a great opportunity for a new contributor's first PR. Should we add a label/tag to make them easier to find? :)

**Reply by AucaCoyan** [reply]
Right! I created an issue from this discussion 😄 . #1766
